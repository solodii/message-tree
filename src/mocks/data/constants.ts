export const NONEXISTENT_MESSAGE = 'NONEXISTENT_MESSAGE';
export const NONEXISTENT_SUBJECT = 'NONEXISTENT_SUBJECT';
