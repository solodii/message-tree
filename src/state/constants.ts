export const ROOT_SUBJECT = '';
