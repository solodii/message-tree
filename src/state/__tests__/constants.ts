import * as constants from '../constants';

test('constants are correct', () => {
  expect(constants).toMatchSnapshot();
});
