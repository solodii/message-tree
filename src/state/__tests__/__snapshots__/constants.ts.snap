// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants are correct 1`] = `
Object {
  "ROOT_SUBJECT": "",
}
`;
